### DISCUSSION TITLE
How to connect mq135 to adc pin of air103?

### DISCUSSION THREAD

**Discussion by jokokiswan** [openLuat/LuatOS#74] [Q&A] [answered]
Does it need to do pull up or pull down for the sensor pin? Since it used 5v system. Schematic for connecting basic sensor are need for newbie like me, thank you luat os team!

**Comment by chenxuuu** [comment, marked as answer]
你可能需要使用分压电阻，因为air103的adc范围是0-2.4v


You may need to use divider resistance because the ADC range of the AIR103 is 0-2.4V.



---Original---
From: ***@***.***&gt;
Date: Sun, Nov 27, 2022 21:08 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [openLuat/LuatOS] How to connect mq135 to adc pin of air103?(Discussion #74)




 
Does it need to do pull up or pull down for the sensor pin? Since it used 5v system. Schematic for connecting basic sensor are need for newbie like me, thank you luat os team!

**Reply by jokokiswan** [reply]
Ok, thank you :)
